packages/perseus/src/util/graphie.ts

```diff
@@ -950,16 +951,11 @@ export class Graphie {
             const pad = this.currentStyle["label-distance"];
 
             $span
-                .css(
-                    $.extend(
-                        {},
-                        {
-                            position: "absolute",
-                            padding: (pad != null ? pad : 7) + "px",
-                            color: "black",
-                        },
-                    ),
```

Comment:
Weird...extending an empty object. That'd make sense if we wanted to force a copy but I don't understand why this would ever be needed.

Reply:
Yeah, the intent is to copy the object, but it's not necessary here, since there are no other references to the object we're copying. I suspect this is copy-pasta from some other code where we were updating `currentStyle` and wanted to do so without mutating the style object.